dashboard/app/main.go

```diff
@@ -1938,6 +1939,7 @@ func createUIBug(c context.Context, bug *Bug, state *ReportingState, managers []
 	uiBug := &uiBug{
 		Namespace:      bug.Namespace,
 		Title:          bug.displayTitle(),
+		ImpactScore:    TitleToImpact(bug.Title),
```

Comment:
I think we at least should look at AltTitles.

What do you think about duplicates? If I duped KASAN write to a WARNING, should it have WARNING score?